net/gallery-dl/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                gallery-dl
+version             1.19.3
+categories          net
+maintainers         nomaintainer
+revision            0
+
+checksums           rmd160  a13818aa36cced652611d66046bed81b65285637 \
+                    sha256  db3973a17f1074e4f4cb99b635b94c1926bdd549263e1df909498f3c6aa93484 \
+                    size    389787
+
+description         command-line program to download image galleries and collections from several image hosting sites
+long_description    ${name} is a {*}${description} It is a cross-platform tool with many configuration options and powerful filenaming capabilities.
+
+homepage            https://github.com/mikf/gallery-dl
+platforms           darwin
+supported_archs     noarch
+license             GPL-2
+
+github.setup        mikf gallery-dl 1.19.3 v
```

Comment:
if you're using the github PG then you shouldn't define `name`, `version`, `homepage`, and probably `distname` either - these are all deduced from the `github.setup` line